### PR TITLE
AUDIO: Fix compiler warning

### PR DESCRIPTION
**Pull request by orgads**
size_t and long are not the same on Win64.

### CODE REVIEW

File: audio/soundfont/vgmcoll.cpp [hunk 1]
```diff
@@ -185,7 +185,7 @@ SynthFile *VGMColl::CreateSynthFile(VGMInstrSet *theInstrSet) {
 				newRgn->SetWaveLinkInfo(0, 0, 1, (uint32) realSampNum);
 
 				if (realSampNum >= finalSamps.size()) {
-					debug("Sample %lu does not exist", realSampNum);
+					debug("Sample %u does not exist", unsigned(realSampNum));
```
Comment (bluegr) [inline audio/soundfont/vgmcoll.cpp:188]:
```suggestion
					debug("Sample %u does not exist", (uint32) realSampNum);
```

Comment (orgads) [reply audio/soundfont/vgmcoll.cpp:188]: I agree that `uint32` is more portable than `unsigned`, but why do you prefer C-style cast over ctor-like cast, which is usually a better choice for C++?

Comment (bluegr) [reply audio/soundfont/vgmcoll.cpp:188]: In this case, mostly for portability reasons, as well as uniformity with the rest of the code

**Comment by orgads** [comment]
Thanks for merging and fixing, and sorry for the delay. I was on vacation.

**Pull request merged**